packages/now-static-build/src/utils/_shared.ts

```diff
@@ -0,0 +1,40 @@
+import { PackageJson } from '@vercel/build-utils';
```

Comment:
Any specific reason for the underscore at the start of the file name? I don't think we have this convention anywhere else in this package.

Reply:
No specific reason. I could change it if we want, but I'd prefer not to re-trigger CI for that change.